ui/hidden_fast_forward_button.js

```diff
@@ -0,0 +1,118 @@
+/*! @license
+ * Shaka Player
+ * Copyright 2016 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+goog.provide('shaka.ui.HiddenFastForwardButton');
+
+goog.require('shaka.ui.Element');
+goog.require('shaka.util.Timer');
+goog.require('shaka.ui.Enums');
+goog.require('shaka.util.Dom');
+
+goog.requireType('shaka.ui.Controls');
+
+/**
+ * @extends {shaka.ui.Element}
+ * @final
+ * @export
+ */
+shaka.ui.HiddenFastForwardButton = class extends shaka.ui.Element {
```

Comment:
This and the other button are almost the exact same code-wise. You could probably make a super-class that handles the basic functionality, and just have these classes pass in a few values: the icon code (FAST_FORWARD or REWIND), the step size (+5 or -5), and the aria label.